src/Rules/PHPUnit/AssertSameWithCountRule.php

```diff
@@ -36,10 +36,13 @@ public function processNode(Node $node, Scope $scope): array
 		if (
 			$right instanceof Node\Expr\FuncCall
 			&& $right->name instanceof Node\Name
-			&& strtolower($right->name->toString()) === 'count'
+			&& in_array(strtolower($right->name->toString()), ['count', 'sizeof'])
```

Comment:
@ondrejmirtes Should I assigned the `functionName` inside the if, or is fine to repeat `strtolower($right->name->toString()`?